src/components/htmlMediaHelper.js

```diff
@@ -31,7 +31,7 @@ define(['appSettings', 'browser', 'events'], function (appSettings, browser, eve
     }
 
     function enableHlsShakaPlayer(item, mediaSource, mediaType) {
-
+        /* eslint-disable-next-line compat/compat */
```

Comment:
window.MediaSource is not available on Chrome 27. As the feature is conditional anyway, I disabled the compat check to prevent ESLint from throwing an error.